src/ralph/ui/forms/addresses.py

```diff
@@ -388,6 +389,9 @@ def __init__(self, *args, **kwargs):
     widget = IPWithHostWidget()
 
     def compress(self, value):
+        hostname, ip_number = value
+        if not ip_number:
+            raise ValidationError(_('IP Address is required'))
```

Comment:
I know this works, but more reasonable place for this code is clean(..), I think.